utils/index.sh

```diff
@@ -197,6 +197,15 @@ get_architecture_config(){
     fi
 }
 
+get_stream(){
```

Comment:
Did you consider using [ADDITIONAL_PARAMS](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/fc43dbff80361e59d812e7c218620f535db6f691/utils/index.sh#L264) for this one from openshift CI itself? Because this is not modifying any existing field and just going about adding one, we need to be mindful about adding more stuff here which is not relevant for all the runs.

Reply:
Good point. The problem with orion filtering you mentioned will anyways exist regardless of adding/updating an existing field. So added it as a planned enhancement to [orion improvements document](https://docs.google.com/document/d/1Cs_GWUKvdinBFze1SmrJHCcqAC_MbEPsjLeMACekuBU/edit?tab=t.0).

And w.r.t stream field, I will let you make that call on its importance for being added as new field in metadata. I would go in the direction as treating it as an additional parameter and having an enhancement in orion to also be able to apply `not match` in the metadata based filtering step.